test/e2e/util/util.go

```diff
@@ -19,6 +21,34 @@ func PrepareE2eEnvironment() *TestE2eConfig {
 
 // initE2E get environment variables and init e2e environment.
 func initE2E() *TestE2eConfig {
+
+	pwd, err := os.Getwd()
+	Expect(err).To(BeNil())
+	projectName := path.Base(path.Clean(path.Join(pwd, "..", "..", "..")))
+	if v := os.Getenv("HUB_NAME"); v == "" {
+		os.Setenv("HUB_NAME", projectName+"-e2e-test-hub")
+	}
+	if v := os.Getenv("HUB_CTX"); v == "" {
+		os.Setenv("HUB_CTX", "kind-"+os.Getenv("HUB_NAME"))
+	}
+	if v := os.Getenv("MANAGED_CLUSTER1_NAME"); v == "" {
+		os.Setenv("MANAGED_CLUSTER1_NAME", projectName+"-e2e-test-c1")
+	}
+	if v := os.Getenv("MANAGED_CLUSTER1_CTX"); v == "" {
+		os.Setenv("MANAGED_CLUSTER1_CTX", "kind-"+os.Getenv("MANAGED_CLUSTER1_NAME"))
+	}
+	if v := os.Getenv("MANAGED_CLUSTER2_NAME"); v == "" {
+		os.Setenv("MANAGED_CLUSTER2_NAME", projectName+"-e2e-test-c2")
+	}
+	if v := os.Getenv("MANAGED_CLUSTER2_CTX"); v == "" {
+		os.Setenv("MANAGED_CLUSTER2_CTX", "kind-"+os.Getenv("MANAGED_CLUSTER2_NAME"))
+	}
+
+	home, err := os.UserHomeDir()
+	Expect(err).To(BeNil())
+	os.Setenv("KUBECONFIG", filepath.Join(home,".kube","config"))
```

Comment:
already set env in makefile https://github.com/open-cluster-management-io/clusteradm/blob/main/test/e2e/e2e-test.mk#L2-L10